benchmark/code/minimatch.js

```diff
@@ -1,12 +1,11 @@
 'use strict';
 
-
 var minimatch = require('minimatch').match;
 var multimatch = require('multimatch');
 
 module.exports = function (files, patterns, options) {
   if (Array.isArray(patterns)) {
-    return multimatch(files, patterns, options);
+    return multimatch.apply(null, arguments);
```

Comment:
I just split the multimatch benchmarks out from minimatch https://github.com/jonschlinkert/micromatch/commit/92664ae68d61eec993ec98ff55764c09f20d212b. when I made the changes I was thinking it made more sense to apply args.